src/model.cpp

```diff
@@ -19,6 +19,8 @@ limitations under the License.Some license of other
 
 namespace libcellml {
 
+static const std::string initial_name = "OhOXiRmNswYoVBA2InM7NRPvQ0HMXFR2wNhoFExd";
```

Comment:
Why this? Why if someone was to use exactly that name for a model? Why not simply have an empty string?